### PR TITLE
Update display composable mapview sample

### PR DESCRIPTION
**Pull request by ruiqima**
## Description

PR to revisit sample to reflect new changes in Kotlin toolkit.

## Links and Data

#3586

## What To Review

- [ ] Build and run the sample to check for any errors.
- [ ] Update the MapViewModel to be able to work with the geo-compose module.
- [ ] Add the line to the "Additional information section": This sample uses the GeoCompose Toolkit module to be able to implement a Composable MapView.
- [ ] Replace the application parameter from the MainScreen with the use of LocalContext.current.applicationContext 
- [x] Add the tags: geocompose, tookit
- [x] Update the sample's metadata.json file to reflect file and tags changes

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

**Comment by shubham7109** [comment]
Could you update the `README.metadata.json` to resolve the GitHub checker [error](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/actions/runs/8146716501/job/22265808821)?

File: display-composable-mapview/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Display Composable MapView
+# Display composable mapview
```
Comment (shubham7109) [inline display-composable-mapview/README.md:1]: Let's not use lowercase here, I realize this is outside of the conversion used in the sample names. Both `Composable` and `MapView` are product-API names and should be okay to use for this sample:  

 
```suggestion
# Display Composable MapView
```

File: display-composable-mapview/README.metadata.json [hunk 2]
```diff
@@ -23,9 +25,7 @@
         "MapView"
     ],
     "snippets": [
-        "src/main/java/com/esri/arcgismaps/sample/displaycomposablemapview/MainActivity.kt",
-        "src/main/java/com/esri/arcgismaps/sample/displaycomposablemapview/MapViewWithCompose.kt",
-        "src/main/java/com/esri/arcgismaps/sample/displaycomposablemapview/theme/Theme.kt"
+        "src/main/java/com/esri/arcgismaps/sample/displaycomposablemapview/MainActivity.kt"
     ],
-    "title": "Display Composable MapView"
+    "title": "Display composable mapview"
```
Comment (shubham7109) [inline display-composable-mapview/README.metadata.json:30]: Same comment here

**Review by shubham7109 — APPROVED** [review]
Looks good to approve 👍

**Review by prupani-7 — APPROVED** [review]
LGTM

**Pull request merged**